jax/example_libraries/optimizers.py

```diff
@@ -12,7 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Optimizers for use with JAX.
+"""Examples of how to write optimizers with JAX.
+
+This is probably not what you are looking for! The optimizers in this library
```

Comment:
Thoughts on this tweak?
```suggestion
You likely do not mean to import this module! The optimizers in this library
```

Reply:
Done.